### PR TITLE
[bitnami/kibana] Add Kibana Helm Chart tests

### PR DESCRIPTION
**Pull request by superaleks**
**Description of the change**

**Test results**

There have been 3 consecutive executions with no failures which are equal to green 9 executions in total. These are the results of the last execution. 

- [GKE](https://cp.production.k.bromelia.bitnami.net/v1/execution-graphs/0607cd6f-39f2-4bbb-8398-77b4db7d00fa/report) - PASS
- [AKS](https://cp.production.k.bromelia.bitnami.net/v1/execution-graphs/0e65797e-2fb1-4b82-81e9-c54acb1c50eb/report) - PASS
- [TKG](https://cp.production.k.bromelia.bitnami.net/v1/execution-graphs/3d67e5bf-2cbd-4556-bc2f-7dcf14517bce/report) - PASS

Adding the tests for the Kibana Helm chart. 

**Benefits**

When someone makes a change in the Helm Chart, they will be able to check if their change has broken the Helm chart by running these automated tests.

**Possible drawbacks**

With UI tests, there is always a possibility for flakiness. This has been mitigated as much as possible.

**Checklist**

- [X] All commits signed off and in the agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: .vib/kibana/cypress/cypress/integration/kibana_spec.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  cy.get('a[data-test-subj="uploadFile"]').should('be.visible').click();
+  cy.get('input#filePicker').selectFile('cypress/fixtures/test-data-to-import.ndjson', {force:true});
+  cy.get('[data-test-subj="dataVisualizerFileOpenImportPageButton"]',).should('be.visible').click();
+  cy.fixture('testdata').then((td) => {
+    cy.get('[data-test-subj="dataVisualizerFileIndexNameInput"]').type(`${td.testIndexName}.${random}`);
+  });
+  cy.get('[data-test-subj="dataVisualizerFileImportButton"]').should('be.visible').click();
+  cy.get('[title="Step 2: Index created is complete"]').scrollIntoView().should('be.visible');
+  cy.get('[title="Step 3: Data uploaded is complete"').scrollIntoView().should('be.visible');
+  cy.get('[title="Step 4: Index pattern created is complete"]').should('be.visible');
```
Comment (joancafom) [inline .vib/kibana/cypress/cypress/integration/kibana_spec.js:23]: If possible, you can use `[title*="Index created is complete"]` instead. This is somewhat more generic and will work even if the step number is modified in upcoming versions.

File: .vib/kibana/cypress/cypress/integration/kibana_spec.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  })
+})
+
+it('allows creating a visualisation', () => {
+  cy.visit('app/visualize/');
+  closeThePopups();
+  cy.contains('button','Create').should('be.visible').click();
+  cy.get('[data-test-subj="visType-vega"]').should('contain.text','Custom visualization').forceClick();
+  cy.contains('[data-test-subj="visualizeSaveButton"]','Save').should('exist').forceClick();
+  cy.get('[data-test-subj="savedObjectTitle"]').should('exist').type('Test visual');
```
[the remark this answers is not in the document]
Comment (superaleks) [reply .vib/kibana/cypress/cypress/integration/kibana_spec.js:50]: Oh, I simply forgot to add this to the fixture folder. Indeed.

File: .vib/kibana/cypress/cypress/integration/kibana_spec.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  closeThePopups();
+  cy.get('[data-test-subj="create-workpad-button"]')
+    .should('be.visible')
+    .forceClick();
+  cy.get('[data-test-subj="add-element-button"]')
+    .should('have.text', 'Add element')
+    .forceClick();
+  cy.contains('span', 'Text').forceClick();
+  cy.visit('app/canvas#/');
+  cy.contains('My Canvas Workpad').should('be.visible');
```
[the remark this answers is not in the document]
Comment (superaleks) [reply .vib/kibana/cypress/cypress/integration/kibana_spec.js:136]: Okay, I can make more changes here like adding a custom title. The name for canvas can be repeated many times, which is exactly what happens.

**Review by joancafom — APPROVED** [review]
LGTM

**Pull request merged**